Rubberduck.Parsing/Grammar/VBALexer.g4

```diff
@@ -295,8 +295,7 @@ SINGLEQUOTE : '\'';
 UNDERSCORE : '_';
 WS : [ \t];
 GUIDLITERAL : '{' [0-9A-F]+ '-' [0-9A-F]+ '-' [0-9A-F]+ '-' [0-9A-F]+ '-' [0-9A-F]+ '}';
-LETTER_RANGE : LETTER WS? MINUS WS? LETTER;
-IDENTIFIER :  ~[[\]()\r\n\t.,'"|!@#$%^&*\-+:=; 0-9-/\\] ~[[\]()\r\n\t.,'"|!@#$%^&*\-+:=; ]*;
+IDENTIFIER :  ~[[\]()\r\n\t.,'"|!@#$%^&*\-+:=; 0-9-/\\-] ~[[\]()\r\n\t.,'"|!@#$%^&*\-+:=; -]*;
```

Comment:
Hmm, we added a hyphen at the end of the subexpression `[[\]()\r\n\t.,'"|!@#$%^&*\-+:=; 0-9-/\\-]` but we did not remove one of the two original hyphens. Should we remove the `\-` from the `...^&*+\-+=;` part? Even if ANTLR is interpreting the hyphen as a character class `\-+`, this doesn't make sense because `+` is 0x2B while `\` is 0X5C.